### ISSUE TITLE
Image Classification not recorded in Docker

### ISSUE THREAD

**Issue by rgaiacs** [heartexlabs/label-studio#1791]
**Describe the bug**

Some times, Label Studio doesn't record the image classification after some images.

**To Reproduce**

1. Launch Label Studio from Docker
2. Create an Image Classification project with images from list of local URL
3. Classify some images
4. Reopen the project

**Expected behavior**

Image classification is save.

**Environment (please complete the following information):**
 - OS: Ubuntu
 - Label Studio Version: 1.3
 - Browser: Firefox (Snap)

**Additional context**

Label Studio report the image classification saved. The HTTP request return 201.

**Comment by makseq** [comment]
@rgaiacs Could you please record a video with this weird bug? Please, open the browser console with errors before you start.

**Comment by rgaiacs** [comment]
@makseq I recorded a video: https://www.youtube.com/watch?v=h_nRKsdMNFQ The video description has timestamps for key parts of the video. Based on the video, the bug is that Label Studio at a given moment does **not** list some of the tasks. When user reload the task list, the missing tasks are listed.

The log from the Docker container showing the jump from 8400 to 8365 in the saved tasks is

```
labelstudio_1  | [30/Nov/2021 04:09:28] "GET /api/dm/tasks/8402?project=9 HTTP/1.1" 200 962
labelstudio_1  | [30/Nov/2021 04:09:34] "GET /api/dm/tasks/8401?project=9 HTTP/1.1" 200 427
labelstudio_1  | [30/Nov/2021 04:09:36] "POST /api/tasks/8401/annotations?project=9 HTTP/1.1" 201 398
labelstudio_1  | [30/Nov/2021 04:09:36] "GET /api/dm/tasks/8401?project=9 HTTP/1.1" 200 962
labelstudio_1  | [30/Nov/2021 04:09:41] "GET /api/dm/tasks/8400?project=9 HTTP/1.1" 200 427
labelstudio_1  | [30/Nov/2021 04:09:45] "POST /api/tasks/8400/annotations?project=9 HTTP/1.1" 201 281
labelstudio_1  | [30/Nov/2021 04:09:45] "GET /api/dm/tasks/8400?project=9 HTTP/1.1" 200 740
labelstudio_1  | [30/Nov/2021 04:09:52] "GET /api/dm/tasks/8365?project=9 HTTP/1.1" 200 427
labelstudio_1  | [30/Nov/2021 04:09:55] "POST /api/tasks/8365/annotations?project=9 HTTP/1.1" 201 398
labelstudio_1  | [30/Nov/2021 04:09:55] "GET /api/dm/tasks/8365?project=9 HTTP/1.1" 200 962
labelstudio_1  | [30/Nov/2021 04:10:04] "GET /api/dm/tasks/8364?project=9 HTTP/1.1" 200 427
labelstudio_1  | [30/Nov/2021 04:10:08] "POST /api/tasks/8364/annotations?project=9 HTTP/1.1" 201 398
```

The only error in the web browser console is

```
Error: [mobx-state-tree] You are trying to read or write to an object that is no longer part of a state tree. (Object type: 'ImageModel', Path upon death: '/annotationStore/root/children/0/children/0', Subpath: 'initialWidth', Action: '/annotationStore/root/children/0/children/0.setStageRef()'). Either detach nodes first, or don't use objects after removing / replacing them in the tree.
    MobX 7
    setStageRef Image.js:611
    MobX 5
    ref ImageView.js:712
    i ReactKonvaCore.js:49
    yj ReactKonvaCore.js:71
    React 3
    unstable_runWithPriority scheduler.production.min.js:18
    React 4
    unstable_runWithPriority scheduler.production.min.js:18
    React 4
    MobX 9
    selectTask lsf-sdk.js:200
    loadTask lsf-sdk.js:183
    submitCurrentAnnotation lsf-sdk.js:446
    Vp lsf-sdk.js:308
    invoke events.ts:33
    invoke events.ts:33
    submitAnnotation AppStore.js:424
    n AppStore.js:403
    submitAnnotation AppStore.js:423
    MobX 5
    attachHotkeys AppStore.js:225
    addKey Hotkey.js:80
    37 keymaster.js:107
    37 keymaster.js:268
    r helpers.js:72
    _wrapEventTarget trycatch.js:137
    h instrument.js:476
    y keymaster.js:262
    37 keymaster.js:268
    37 keymaster.js:297
    c main.js:2
    s hot module replacement:36
    6777 main.js:2
    c main.js:2
    <anonymous> jsonp chunk loading:498
    <anonymous> jsonp chunk loading:499
    <anonymous> universalModuleDefinition:9
    <anonymous> universalModuleDefinition:10
```

Thanks for the help!

**Comment by makseq** [comment]
@rgaiacs I see you have some task filter enabled. 
![image](https://user-images.githubusercontent.com/501780/144527026-ebf4ea55-e733-43d6-85cc-eea5c83b55a7.png)
Are you sure it's not the cause of your problem?

**Comment by rgaiacs** [comment]
The filter was configure to show only images without classification. And the filter was all the time enabled.

**Comment by jmilde** [comment]
I can reproduce the problem, but could not find a pattern.
In our case we are updating a previously labeled images.
The problem exists with the docker images: latest, 1.4.0 and 1.3.0.

**Comment by makseq** [comment]
@jmilde please, try to record a video with this problem reproduction.. we couldn't repeat it.

**Comment by jmilde** [comment]
@makseq https://www.youtube.com/watch?v=-Qz6jeaJfns
As you can see it works for some images but not for others (in the video we scrolled to one that we knew causes problems), and sometimes when trying often enough then the update will work.


`Error: [mobx-state-tree] You are trying to read or write to an object that is no longer part of a state tree. (Object type: 'ImageModel', Path upon death: '/annotationStore/root/children/0', Subpath: 'initialWidth', Action: '/annotationStore/root/children/0.setStageRef()'). Either detach nodes first, or don't use objects after removing / replacing them in the tree.
    MobX 7
    setStageRef Image.js:419
    MobX 5
    ref ImageView.js:511
    s ReactKonvaCore.js:49
    oe ReactKonvaCore.js:71
    React 13
    MobX 9
    selectTask lsf-sdk.js:170
    loadTask lsf-sdk.js:149
    Rp lsf-sdk.js:69
    invoke events.ts:32
    invoke events.ts:32
    updateAnnotation AppStore.js:367
    MobX 5
    React 15
    r helpers.js:72
    _wrapEventTarget trycatch.js:137
    h instrument.js:476
    React 8
    createApp LabelStudio.js:53
    LabelStudio LabelStudio.js:30
    initLabelStudio lsf-sdk.js:119
    Rp lsf-sdk.js:114
    initLSF dm-sdk.js:342
    s Label.js:42
    invoke dm-sdk.js:313
    invoke dm-sdk.js:313
    setSelected DataStore.js:43
    MobX 4`

**Comment by makseq** [comment]
@jmilde Thank you very much for this video, looks like a problem with autosaved drafts. I've created a ticket about this.

**Comment by makseq** [comment]
This issue was solved in the latest LS versions (after 1.5).
